var/spack/repos/builtin/packages/gpi-2/package.py

```diff
@@ -3,77 +3,176 @@
         'fabrics',
         values=disjoint_sets(
-            ('auto',), ('verbs',), ('ethernet',),
-        ).with_non_feature_values('auto', 'none'),
+            ('infiniband',), ('ethernet',),
+        ).with_non_feature_values('none'),
         description="List of fabrics that are enabled; "
-        "'auto' lets gpi-2 determine",
+        "'none' use GPI-2 default (infiniband), "
+        "'infiniband' will use 'rdma-core' from Spack",
     )
 
+    depends_on('rdma-core', when='fabrics=infiniband')
+
     variant(
-        'schedulers',
+        'launchers',
         values=disjoint_sets(
-            ('auto',), ('loadleveler',), ('pbs',), ('slurm',)
-        ).with_non_feature_values('auto', 'none'),
-        description="List of schedulers for which support is enabled; "
-        "'auto' lets gpi-2 determine",
+            ('loadleveler',), ('pbs',), ('slurm',)
+        ).with_non_feature_values('none', 'pbs'),
+        description="List of lauchers for which support is enabled; "
+        "'none' or 'pbs' use 'gaspi_run.ssh'",
     )
 
-    depends_on('autoconf', type='build')  # autogen.sh - autoreconf
-    depends_on('automake', type='build')  # autogen.sh - automake
-    depends_on('libtool',  type='build')
-    depends_on('m4',       type='build')
-    depends_on('mpi', when='+mpi')
+    depends_on('slurm', when='launchers=slurm')
 
-    depends_on('rdma-core', when='fabrics=verbs')
+    def set_specific_cflags(self, spec):
+        if spec.satisfies('@:1.3.2%gcc@10.1.0:'):
+            raise RuntimeError('Gcc version must be less than 10.1')
+        elif spec.satisfies('@:1.4.0%gcc@10.1.0:'):
+            environ['CFLAGS'] = '-fcommon'
 
-    depends_on('slurm', when='schedulers=slurm')
+    # GPI-2 without autotools
+    @when('@:1.3.2')
+    def autoreconf(self, spec, prefix):
+        touch = which('touch')
+        touch('configure')
+        pass
 
-    def with_or_without_verbs(self, activated):
-        opt = 'infiniband'
-        # If the option has not been activated return
-        # --without-infiniband
-        if not activated:
-            return '--without-{0}'.format(opt)
-        return '--with-{0}={1}'.format(opt, self.spec['rdma-core'].prefix)
+    @when('@:1.3.2')
+    def configure(self, spec, prefix):
+        pass
+
+    @when('@:1.3.2')
+    def build(self, spec, prefix):
+        self.old_install(spec, prefix)
+        pass
+
+    @when('@:1.3.2')
+    def old_install(self, spec, prefix):
+        spec = self.spec
 
+        self.set_specific_cflags(spec)
+
+        config_args = ['-p {0}'.format(prefix)]
+        if 'fabrics=ethernet' in spec:
+            config_args += ['--with-ethernet']
+        elif 'fabrics=infiniband' in spec:
+            config_args += ['--with-infiniband={0}'.format(spec['rdma-core'].prefix)]
+        if 'launchers=loadleveler' in spec:
+            config_args += ['--with-ll']
+        elif 'launchers=slurm' in spec:
+            raise RuntimeError('Slurm is not supported')
```

Comment:
Should be a `conflicts` rather than `RuntimeError`.

Reply:
Done as a conflict